css-three-column-layout/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>
+          Designed for life science. Optimized for SaMD and Digital Health.
+        </h1>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third column-container">
+          <div class="img-container">
+            <div>
+              <img src="images/merge.png" alt="merge" />
+            </div>
+          </div>
+          <h2>Streamlined QMS</h2>
+          <p>
+            A QMS that includes modules for all stages of the product life cycle
+            from Design Control to CAPA
+          </p>
+        </div>
+
+        <div class="column-one-third column-container">
+          <div class="img-container">
+            <div>
+              <img src="images/cloud-computing.png" alt="cloud computing" />
+            </div>
```

Comment:
You should try to make `alt` attributes more descriptive. These are used for both in the image is broken and for vision assistance/screen readers. Instead of just saying "Cloud computing" this would be something like "Cloud Computer Icon" or "Icon connecting computer to cloud". The others could be "Icon of people having a discussion" and "Icon of paths merging together"